packages/salesforcedx-vscode-replay-debugger/src/constants.ts

```diff
@@ -25,7 +24,10 @@ export const EVENT_VARIABLE_SCOPE_BEGIN = 'VARIABLE_SCOPE_BEGIN';
 export const EVENT_VARIABLE_ASSIGNMENT = 'VARIABLE_ASSIGNMENT';
 export const EXEC_ANON_SIGNATURE = 'execute_anonymous_apex';
 export const FIELD_INTEGRITY_EXCEPTION = 'FIELD_INTEGRITY_EXCEPTION';
+export const INVALID_CROSS_REFERENCE_KEY = 'INVALID_CROSS_REFERENCE_KEY';
 export const GET_LINE_BREAKPOINT_INFO_EVENT = 'getLineBreakpointInfo';
 export const LINE_BREAKPOINT_INFO_REQUEST = 'lineBreakpointInfo';
+export const OVERLAY_ACTION_DELETE_URL =
+  'services/data/v43.0/tooling/sobjects/ApexExecutionOverlayAction/';
```

Comment:
It would be nice if this was used in the tests as well.

Reply:
Thanks for kindly pointing that out to me. Done.